scripts/apps/search/MultiImageEdit.ts

```diff
@@ -204,6 +206,32 @@ export function MultiImageEditDirective(asset, $sce) {
                     scope.selectImage(image);
                 }
             };
+
+            scope.copyMetadata = (metadata) => {
+                scope.metadataFromStorage = metadata;
+                localStorage.setItem('metadata:items', JSON.stringify(metadata));
+            };
+
+            scope.toggleAutofill = () => {
+                scope.autoFill = !scope.autoFill;
+                localStorage.setItem('metadata:autofill', scope.autoFill);
+            };
+
+            scope.pasteMetadata = () => {
+                scope.metadata = JSON.parse(localStorage.getItem('metadata:items'));
```

Comment:
you have `scope.metadataFromStorage` defined earlier, why not use it here? or why load it before?

Reply:
I changed scope.metadataFromStorage to be just boolean, so now I use it just for checks